qa/tasks/cephfs/caps_helper.py

```diff
@@ -90,9 +90,23 @@ def _gen_mds_cap_str(perm, fsname=None, cephfs_mntpt='/'):
     return mds_cap
 
 
-# XXX: Inheriting CephFSTestCase ONLY to get methods like self.assertEqual(),
-# self.assertIn(), etc.
-class CapTester(CephFSTestCase):
+def assert_equal(first, second):
+    msg = 'Variables are not equal.\nfirst = {first}\nsecond = {second}'
```

Comment:
Doesn't this needs to be formatted (`f`)?

Reply:
Fixed.